src/components/Modal/Modal.vue

```diff
@@ -10,8 +10,8 @@
         >
             <div class="modal-content">
                 <div
-                    v-if="!!$slots.header || !!$slots.title || !!title"
-                    class="modal-header"
+                    v-if="!!$slots.header || !!$slots.title || !!title || !hideCloseBtn"
```

Comment:
![where_test](https://user-images.githubusercontent.com/45201651/157897378-cdc9bc81-7b88-4684-82fc-bde797bba614.jpg)

Reply:
the test is in the updated snapshot. now by default the btn-close is there